core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -911,4 +911,42 @@ class PlaintextConsumerTest extends BaseConsumerTest {
 
     assertThrows(classOf[WakeupException], () => consumer.position(topicPartition, Duration.ofSeconds(100)))
   }
+
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersConsumerGroupProtocolOnly"))
```

Comment:
Why not for all combinations? I would expect this test passes for the classic consumer, failed for the new one before this PR, and pass for all with this PR, is that correct?

Reply:
That was not the behavior I noticed, unfortunately. The existing consumer was very flaky with this test for both the default timeout and a `0` timeout. I played with it for a bit to make sure the test itself wasn't at fault, but couldn't get the existing consumer to pass reliably.

Reply:
uhm interesting, that probably means there is something we don't understand quite well then with the classic consumer here, or maybe we're still missing something with test itself. Let's maybe give it a try after addressing the following comment specific to the test (if still flaky for the classic, ok to not include it and keep the focus on the new consumer, but let's try to include it if it works)

Reply:
I should mention that _on my machine_, the test consistently passes for both the `CLASSIC` and `CONSUMER` group protocols. However, when the test suite was run on the CI pipeline, the test failed for every run of the `CLASSIC` group protocol. I spent the worse part of a day investigating to reproduce the failure, but I couldn't. Frustrated, I threw up my hands and update the test parameters to only run the `CONSUMER` group protocol. So it's hard to know if any of my changes work until they're run on CloudBees 🐝

Reply:
Just saw another test run where all the `CLASSIC` group protocol test instances failed on CI but works locally. I'm not sure how much time to spend on this. Thoughts?

Reply:
Ir order to make progress, I would suggest we file a separate Jira to enable this for the classic, and keep this PR for the new consumer then.

Reply:
KAFKA-17397 has been filed to capture the test failure for the `ClassicKafkaConsumer`.